adflow/pyADflow.py

```diff
@@ -93,6 +93,12 @@ def __init__(self, comm=None, options=None, debug=False, dtype="d"):
             curDir = os.path.basename(os.path.dirname(os.path.realpath(__file__)))
             self.adflow = MExt.MExt("libadflow", curDir, debug=debug)._module
 
+        # This is a flag not exposed to the user as an option that controls
+        # printing the boundary condition warnings.  This should only
+        # be used in the MPhys layer when we make multiple calls to
+        # updateBCAllLevels.
+        self.adflow.inputiteration.printbcwarnings = True
+
```

Comment:
Same here, this should not be here, like in PR #307. See [here](https://github.com/mdolab/adflow/pull/307#pullrequestreview-1664677955) for original comment.